jboss-cert-helper/main.go

```diff
@@ -1,99 +1,99 @@
 package main
 
 import (
-  "errors"
-  "fmt"
-  "log"
-  "os"
-  "os/user"
-  "syscall"
+	"errors"
```

Comment:
Why has format changed ? This is not pertinent in this PR

Reply:
`go fmt` is the way to go. So it should be tabs, not spaces 😉

Reply:
go fmt appears is a no-op on my machine - so I think this is correct.

Reply:
Yes, `go fmt` from `jboss-cert-helper` and you're good to go.